sky/clouds/aws.py

```diff
@@ -799,7 +799,7 @@ def _get_disk_specs(
             disk_tier: Optional[resources_utils.DiskTier]) -> Dict[str, Any]:
         tier = cls._translate_disk_tier(disk_tier)
         tier2iops = {
-            resources_utils.DiskTier.HIGH: 7000,
+            resources_utils.DiskTier.HIGH: 16000,
```

Comment:
The previous IOPS was selected to keep the `HIGH` disk tier's performance for all clouds similar to each other. Should we instead create a new `DiskTier.ULTRA` for the max possible IOPS?

Reply:
I see. How about we change DiskTier.BEST to the actual best possible disk tier?

Reply:
Right now, `BEST` is the best DiskTier among `LOW`, `MEDIUM`, `HIGH` provided by SkyPilot right now (some clouds may not have `HIGH` tier, so `BEST` will map to `MEDIUM` in that case).

I am fine with changing the semantics of `BEST` to the actual best disk available on a specific cloud. This is related to #3585 and #3517.